frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -925,7 +953,7 @@ export const funnelLogic = kea<funnelLogicType>({
         allProperties: [
             (s) => [s.inversePropertyNames, s.excludedPropertyNames],
             (inversePropertyNames, excludedPropertyNames): string[] => {
-                return inversePropertyNames(excludedPropertyNames)
+                return inversePropertyNames(excludedPropertyNames || [])
```

Comment:
I'm kinda confused how this is different from `propertyNames` now O.o

Reply:
Yeah I think I'm a bit confused as well. Looking at the sequence, as the code is now:

 1. initial correlation page visit, excludedPropertyNames is the default list, we only show all properties minus the excluded properties
 2. you uncheck a property in the dropdown, it updates the propertyNames, it triggers a refresh of correlations, doesn't add to the project level exclude.
 3. you refresh, your selection is void and you're at the same place at step 1
 4. you now click exclude from project
 5. it adds to the excludedPropertyNames, which triggers an update of correlations
 6. you refresh, you 1. don't see the excluded property in the property list, 2. don't see the excluded property in the correlations list

Were you thinking of something different?

Reply:
Ah, this makes sense, yes, that's how it's supposed to be. 🤦‍♂️